integration/containerd/cri/integration-tests.sh

```diff
@@ -180,7 +180,7 @@ check_daemon_setup() {
 		FOCUS="TestImageLoad" \
 		RUNTIME="" \
 		CONTAINERD_CONFIG_FILE="$CONTAINERD_CONFIG_FILE" \
-		make -e cri-integration
+		make GO_BUILDTAGS="no_btrfs" -e cri-integration
```

Comment:
Mind to leave a inline comment explaining why this build flag is required?

Reply:
Added a comment befor make command, in AlibabaCloud there is no btrfs package available.